packages/components/src/internal/query/filter.ts

```diff
@@ -192,15 +192,20 @@ export function getDateFieldLabKeySql(filter: Filter.IFilter, tableAlias?: strin
     return null;
 }
 
-function getInClauseLabKeySql(filter: Filter.IFilter, jsonType: JsonType, tableAlias?: string): string {
+function getInClauseLabKeySql(
+    filter: Filter.IFilter,
+    jsonType: JsonType,
+    tableAlias?: string,
+    noNegate?: boolean
+): string {
     const filterType = filter.getFilterType();
     const columnNameSelect = getLegalIdentifier(filter.getColumnName(), tableAlias);
     let operatorSql = null;
 
     const values = filterType.parseValue(filter.getValue());
 
     const sqlValues = [];
-    const negate = filterType.getURLSuffix() === Filter.Types.NOT_IN.getURLSuffix();
+    const negate = filterType.getURLSuffix() === Filter.Types.NOT_IN.getURLSuffix() && !noNegate;
```

Comment:
A triple negative! Can we be more positive somehow?  The `flipNegative` from the calling methods reads better to me.